keyboards/ergotravel/keymaps/yanfali/rules.mk

```
@@ -1 +1,7 @@
 BOOTLOADER = qmk-dfu
+
+BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration(+1000)
+CONSOLE_ENABLE = yes         # Console for debug(+400)
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
+LINK_TIME_OPTIMIZATION_ENABLE = yes
```

[Optional/FYI] This could be:

```suggestion
LTO_ENABLE = yes
```